src/physics/mhd.cpp

```diff
@@ -324,42 +324,6 @@ ::convective_flux (const std::array<real,nstate> &conservative_soln) const
     return conv_flux;
 }
 
-template <int dim, int nstate, typename real>
-std::array<dealii::Tensor<1,dim,real>,nstate> MHD<dim, nstate, real>
-::convective_numerical_split_flux(const std::array<real,nstate> &soln_const,
```

Comment:
Is this function in MHD being replaced by something else?

Reply:
it was put there as a placeholder -- MHD never had an implementation for it